benchmarks/src/lib/constants/benchmark-tests.ts

```diff
@@ -1,403 +1,640 @@
-import { MathOperator, Method } from '@joshdb/provider';
+import { MathOperator, Method, Payload } from '@joshdb/provider';
 import type { Benchmark } from '../Benchmark';
 
 export const BENCHMARK_TESTS: Benchmark.Test[] = [
   {
     name: Method.AutoKey,
 
     run: async ({ josh }) => {
-      await josh.autoKey();
+      await josh.autoKey({ method: Method.AutoKey, errors: [] });
     }
   },
   {
     name: Method.Clear,
 
     beforeEach: async ({ josh, entries }) => {
-      await josh.setMany(entries);
+      await josh.setMany({
+        method: Method.SetMany,
+        errors: [],
+        overwrite: true,
+        entries: entries.map(([key, value]) => {
+          return { key, value, path: [] };
+        })
+      });
     },
 
     run: async ({ josh }) => {
-      await josh.clear();
+      await josh.clear({ method: Method.Clear, errors: [] });
     }
   },
   {
     name: Method.Dec,
 
     beforeAll: async ({ josh, entries }) => {
-      await josh.setMany(entries);
+      await josh.setMany({
+        method: Method.SetMany,
+        errors: [],
+        overwrite: true,
+        entries: entries.map(([key, value]) => {
```

Comment:
Does converting this hit perf? If so, consider changing this before the function runs?